zanata-war/src/main/webapp/WEB-INF/layout/language/request_join_modal.xhtml

```diff
@@ -0,0 +1,114 @@
+<div xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:h="http://java.sun.com/jsf/html"
+  xmlns:s="http://jboss.org/schema/seam/taglib"
+  xmlns:rich="http://richfaces.org/rich"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:zanata="http://java.sun.com/jsf/composite/zanata"
+  class="modal" id="joinLanguageDialog" tabindex="-1" role="dialog"
+  aria-labelledby="myModalLabel" aria-hidden="true">
+
+  <script>
+    $(document).ready(function() {
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-emailType')}')).
+        val('#{languageJoinAction.emailType}');
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-subject')}')).
+        val('#{languageJoinAction.subject}');
+      jQuery('#' + jqSelector('#{rich:clientId('request-join-id')}')).val('#{id}');
```

Comment:
Because sendEmail is a generic bean for email actions and its used in contactAdmin(which is in everypage), as well as contact coordinator, request to join action. To handle these actions, we only set certain variables (form submit) only when user click "send" button. 

I do agree if we need to refactor sendEmail bean to multiple action handler for each type of email. But then it will be too big change for this.